### PR TITLE
Added shell sort and big integer

### PR DESCRIPTION
**Pull request by RohanGoyal2014**

### CODE REVIEW

**Review by shubhamg931 — APPROVED** [review]
Rejected

**Review by shubhamg931 — CHANGES_REQUESTED** [review]
Please close this PR, and follow hacktoberfest best practices. Such PRs does not maintain the decorum of the event

File: shellSort.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+int main() {
+  int n;
+  cin>>n;
+  int a[n];
+  for(int i=0;i<n;i++)
+  {
+    cin>>a[i];
+  }
+  shell_sort(a,n);
```
Comment (shubhamg931) [inline shellSort.cpp:30]: Why not use sort() function only instead, this is not acceptable

Comment (RohanGoyal2014) [reply shellSort.cpp:30]: Change internal implementation of sort method to use shell sort and then I can do that. Meanwhile resolving this conversation. Awaiting your PR on GNU''s repo which changes sort() method's implementation. ☺️

Comment (shubhamg931) [reply shellSort.cpp:30]: That is just stupid, what are we achieving by wasting efforts on writing our own shell sort here. Where are you using this, please don't make foolish arguments

Comment (RohanGoyal2014) [reply shellSort.cpp:30]: Seems like you are unable to raise a PR for this. No worries. I understand you can't ☺️

Comment (shubhamg931) [reply shellSort.cpp:30]: Why would someone just add a sorting algorithm like this for no reason? At least mention the use case for adding this and some better documentation around it as to why are we using this. This seems like an effort to win hacktoberfest T-shirt, one should not make a habit of such things

Hope you understand :)

**Pull request merged**